python/aws_lambda_powertools/logging/logger.py

```diff
@@ -26,13 +29,17 @@ def logger_setup(service: str = "service_undefined", level: str = "INFO", **kwar
         service name
     LOG_LEVEL: str
         logging level (e.g. INFO, DEBUG)
+    POWERTOOLS_LOGGER_SAMPLE_RATE: str
+        samping rate ranging from 0 to 1, float precision
```

Comment:
Drop ", float precision" and use "1 being 100% sampling" instead. This along with float type annotation should clear out any ambiguity.